### ISSUE TITLE
Request - add institution_acronym to all container-involved bulk tools

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8195]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

See eg https://github.com/ArctosDB/arctos/issues/6789 and linked issues eg https://github.com/ArctosDB/arctos/issues/8194

* I don't think the container environment is fully understood (and most of the time it doesn't need to be)
* Some users seem to have (or had, or ??) access to institutions which might have overlapping barcodes
* Adding an explicit column will hopefully add clarity to the user experience, and the cost would I believe be extremely minimal - for most users most of the time, this would just be a static thing that requires very little consideration.
* Adding an explicit column would remove any chance of an error resulting from me trying to make inferences from user's rights, or tripping up on some change in those rights, or failing in any of the other ways implicit data can  be misunderstood

I'll take this to dev in a week or so if no better ideas or technically-viable alternatives show up here.

**Comment by campmlc** [comment]
I support this. In the case of #8194, the institution is new to Arctos and being set up by previous MSB students and collaborators. There is significant overlap in staff and in samples shared between the institutions. Hence the problem of both institutions using the same barcode series without some sort of guardrails.

**Comment by dustymc** [comment]
See https://github.com/ArctosDB/arctos/issues/8194#issuecomment-2427591009, going dev.
